apps/google-analytics-4/frontend/src/components/config-screen/assign-content-type/AssignContentTypeRow.tsx

```diff
@@ -126,6 +128,13 @@ const AssignContentTypeRow = (props: Props) => {
     );
   };
 
+  useEffect(() => {
+    if (focus) {
+      const contentTypeSelect = document.getElementById(`contentType-${index}`);
+      if (contentTypeSelect) contentTypeSelect.focus();
```

Comment:
Is there a reason to use an explicit `focus()` in the useEffect instead of `tabindex` on the element itself with incremental integer values? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex

Reply:
this element is dynamically injected into the DOM and _above_ the `Add another content type` button. This handler is responsible for setting the focus to the newly added element for optimal/seamless keyboard navigation

Reply:
also, we're not trying to _alter_ the natural tabindex, just set focus to the new element